src/vm.c

```diff
@@ -291,6 +291,16 @@ internal WrenVM* VM_create(ENGINE* engine) {
   MAP_addFunction(&engine->moduleMap, "input", "SystemGamePad.name", GAMEPAD_getName);
   MAP_addFunction(&engine->moduleMap, "input", "SystemGamePad.id", GAMEPAD_getId);
 
+  // Json
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.stream_begin(_)", JSON_streamBegin);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.stream_end()", JSON_streamEnd);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.next", JSON_next);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.value", JSON_value);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.error_message", JSON_error_message);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.lineno", JSON_lineno);
+  MAP_addFunction(&engine->moduleMap, "json", "JsonStream.pos", JSON_pos);
+  MAP_addFunction(&engine->moduleMap, "json", "static JsonStream.escapechar(_,_)", JSON_escapechar);
```

Comment:
The C methods here "belong" to the JsonStream class, so they should be prefixed as `JSON_STREAM_`, rather than `JSON_`.

Reply:
ok